tests/resources/Harbor-Pages/Vulnerability.robot

```diff
@@ -63,6 +63,11 @@ Enable Scan On Push
     Sleep  10
 
 Vulnerability Not Ready Project Hint
-    Page Should Contain Element  //span[contains(@class, "db-status-warning")]
+    ${element}=  Set Variable  xpath=//span[contains(@class, "db-status-warning")]
+    Wait Until Element Is Visible And Enabled  ${element} 
+    Page Should Contain Element  ${element}
```

Comment:
The same with above.

Reply:
yes, you're right, I will remove "Page Should Contain Element".